src/Compilers/CSharp/Portable/Symbols/MemberSymbolExtensions.cs

```diff
@@ -319,6 +318,28 @@ internal static NamespaceOrTypeSymbol OfMinimalArity(this IEnumerable<NamespaceO
             return minAritySymbol;
         }
 
+#nullable enable
+        /// <summary>
+        /// Return all of the type parameters in this symbol and enclosing symbols,
+        /// from outer-most to inner-most type.
+        /// </summary>
+        internal static ImmutableArray<TypeParameterSymbol> GetAllTypeParameters(this Symbol? symbol)
```

Comment:
>GetAllTypeParameters [](http://example.com/codeflow?start=60&length=20)

Is this a replacement of ```TypeSymbolExtensions.GetAllTypeParameters```? If we want to keep that method, should it call this one?